### PR TITLE
Sample Logistic Regression Solution for the Titanic Dataset

### LINKED ISSUE #7: [SIN ]Pre-process the titanic dataset and implement different Machine Learning algorithms
Tasks
- Clean and filter the titanic dataset.
- Implement ML algorithms like
   A few example solutions:
    - Logistic Regression

    - Random Forest Classifier

    - SVM Classifier

    - Decision Tree

    - K-Means

    - Gradient Boosting (XGBoost, CatBoost)

    - Neural networks

    - Other

### PR DESCRIPTION
**Pull request by thewires2**

### CODE REVIEW

**Review by AjayKhalsa — APPROVED** [review]
Looks good

**Comment by thewires2** [comment]
> Looks good

Thanks
